mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -269,13 +263,9 @@ function Data() {
     >
       <Tab key="data" label="Data">
         <Spacing mb={3} mt={3} />
-        <SimpleDataTable
-          columnFlexNumbers={ Array(columnHeaderSample.length).fill(1)}
-          columnHeaders={columnHeaderSample}
-          rowGroupData={rowGroupDataSample && [rowGroupDataSample]}
-        />
+        {dataEl}
```

Comment:
Replacing components with variables seems more helpful when our code is getting super verbose and we wanna clean it up, but i think it can be helpful viewing all the code in one place espec if that component is only 4 lines versus wondering what `dataEl` actually is and then having to go find it.

Reply:
If we are displaying a component twice somewhere, for example, then yeah, that makes sense to use a variable. depends on the situation